demo/src/screens/HomeScreen/HomeCarousel.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Carousel} from 'react-bootstrap';
+import slideOne from '../../images/slide_one.png';
+import slideTwo from '../../images/slide_two.png';
+import slideThree from '../../images/slide_three.png';
+import './HomeCarousel.css';
+
+/**
+ * @return {!JSX} Slideshow component for showcasing
+ *     our latest deals and offerings.
+ */
+export function HomeCarousel() {
+  return (
+    <Carousel>
+      <Carousel.Item>
+        <img
```

Comment:
In another PR I saw using Image from 'react-bootstrap/Image';
Ideally we should stay consistent all over the codebase.

btw, how className would work for the regular "img" tag? (might be my gap in understanding react)

Reply:
I definitely agree we should stay consistent. I'll go ahead and change to Image :)

The className thing is a React weirdness thing - JSX mimics HTML in a lot of ways but is ultimately a flavor of Javascript where class is a reserved word. [Here's some reading if you're curious](https://reactjs.org/docs/dom-elements.html)!

Reply:
Great, thanks for the link!